pkgs/development/libraries/clang-ocl/default.nix

```diff
@@ -34,11 +35,18 @@ stdenv.mkDerivation rec {
     "-DCMAKE_CXX_COMPILER=clang++"
   ];
 
+  passthru.updateScript = writeScript "update.sh" ''
+    #!/usr/bin/env nix-shell
+    #!nix-shell -i bash -p curl jq common-updater-scripts
+    rocmVersion="$(curl -sL "https://api.github.com/repos/RadeonOpenCompute/clang-ocl/releases?per_page=1" | jq '.[0].tag_name | split("-") | .[1]' --raw-output)"
```

Comment:
This should optionally take a GITHUB_TOKEN to not run into rate limiting

Reply:
Like this? https://github.com/NixOS/nixpkgs/blob/350fd0044447ae8712392c6b212a18bdf2433e71/pkgs/applications/emulators/rpcs3/update.sh

Reply:
I was more thinking about something like this https://github.com/NixOS/nixpkgs/blob/b0f8c96b9ef5998f3985c34c90cfd21f1ba8a1ce/pkgs/applications/emulators/citra/update.sh#L30 to make it truly optional

Reply:
Already done in the PR, so we should be good here.